cobalt/media/media_module.cc

```diff
@@ -186,10 +186,9 @@ class CanPlayTypeHandlerStarboard : public CanPlayTypeHandler {
 }  // namespace
 
 bool MediaModule::SetConfiguration(const std::string& name, int32 value) {
-  if (name == "EnableBatchedSampleWrite") {
-    allow_batched_sample_write_ = value;
-    LOG(INFO) << (allow_batched_sample_write_ ? "Enabling" : "Disabling")
-              << " batched sample write.";
+  if (name == "AudioBatchedSampleWrite" && value > 0) {
+    audio_batched_sample_write_ = value;
+    LOG(INFO) << "Set MaximumNumberOfSamplesPerWrite to " << value;
```

Comment:
As suggested by this log message, the setting name used above should be `MaxNumberOfSamplesPerWrite`.

Reply:
Moved to https://github.com/youtube/cobalt/pull/3568.